playground/mocks/data/idp/idx/error-authenticator-duo-verification-timeout.json

```diff
@@ -0,0 +1,145 @@
+{
+  "version": "1.0.0",
+  "stateHandle": "0226IE-haiM0SRi4WOqasDXppYpNa11tsovmA7bvOl",
+  "expiresAt": "2021-05-04T00:28:14.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "challenge-authenticator",
+        "relatesTo": [
+          "$.currentAuthenticatorEnrollment"
+        ],
+        "href": "http://localhost:3000/idp/idx/challenge/answer",
+        "method": "POST",
+        "produces": "application/ion+json; okta-version=1.0.0",
+        "value": [
+          {
+            "name": "credentials",
+            "type": "object",
+            "form": {
+              "value": [
+                {
+                  "name": "signatureData",
+                  "label": "Assertion From Duo",
+                  "required": true,
+                  "visible": false
+                }
+              ]
+            },
+            "required": true
+          },
+          {
+            "name": "stateHandle",
+            "required": true,
+            "value": "0226IE-haiM0SRi4WOqasDXppYpNa11tsovmA7bvOl",
+            "visible": false,
+            "mutable": false
+          }
+        ],
+        "accepts": "application/json; okta-version=1.0.0"
+      }
+    ]
+  },
+  "messages": {
+    "type": "array",
+    "value": [
+      {
+        "message": "We did not hear back from Duo.",
```

Comment:
Is this a real error message from Duo or our API? We should consider changing this to something a bit more formal and with a possible resolution.

Reply:
The server side is sending this message.
`
We were unable to verify with Duo. Try again.
`

https://github.com/okta/okta-core/blob/5750e25bfed06127c8dc6993d0f806777fdf927b/resources/src/main/resources/api-messages.properties#L1004